### PR TITLE
Update ProofEditorPanel.java

### PR DESCRIPTION
**Pull request by wei-jun7**
## Description

Closes #(issue) 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Enhancement (improvement to an already existing feature)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by charlestian23 — CHANGES_REQUESTED** [review]
Please be careful to review your code before you open a pull request. I cannot compile the code due to there being syntax errors.

**Comment by wei-jun7** [comment]
sorry about it I will check it again

**Comment by wei-jun7** [comment]
sorry for the mistake I make and it is fixed now for ctrl + y is redo

File: src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java [hunk 1]
```diff
@@ -293,6 +294,17 @@ public JMenuBar getMenuBar() {
 
         edit.add(fitTreeToScreen);
         fitTreeToScreen.addActionListener((ActionEvent) -> this.fitTreeViewToScreen());
+        {
+            GameBoardFacade.getInstance().getHistory().redo();
+        }
```
Comment (charlestian23) [inline src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java:299]: What do lines 297-299 do?

File: src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java [hunk 2]
```diff
@@ -293,6 +294,17 @@ public JMenuBar getMenuBar() {
...
+        if(os.equals("mac")) {
+            redo.setAccelerator(KeyStroke.getKeyStroke('Z', Toolkit.getDefaultToolkit().getMenuShortcutKeyMask() + InputEvent.SHIFT_DOWN_MASK));
+            redo.setAccelerator(KeyStroke.getKeyStroke('Y', Toolkit.getDefaultToolkit().getMenuShortcutKeyMask() + InputEvent.SHIFT_DOWN_MASK));
```
Comment (charlestian23) [inline src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java:302]: Double check if `InputEvent.SHIFT_DOWN_MASK` should be in line 302. I think that might be doing `cmd+Shift+Y` for Mac users, though I'm not entirely sure.

**Review by charlestian23 — CHANGES_REQUESTED** [review]
For Windows at least, these changes make `Ctrl+Y` work but break the `Ctrl+Shift+Z` functionality.  Double check your code and make sure your thoroughly test it.

**Comment by charlestian23** [comment]
Also please be sure to fill out the checklist when you open a pull request. It makes the reviewer's life a lot easier when reviewing these pull requests. For a sample, you can reference [this](https://github.com/Bram-Hub/Legup/pull/252) pull request.

**Comment by wei-jun7** [comment]
thank you and it is done now

File: src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java (hunk 2 above)
Comment (wei-jun7) [reply src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java:302]: I done have the mac near me and I add the ctrl inside.

File: src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java (hunk 1 above)
Comment (wei-jun7) [reply src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java:299]: no idea

Comment (wei-jun7) [reply src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java:299]: I not change there , I think

Comment (charlestian23) [reply src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java:299]: According to GitHub, those three lines were added by you. I'd remove them from the code to be safe if you're not sure what they do.

**Comment by charlestian23** [comment]
> For Windows at least, these changes make `Ctrl+Y` work but break the `Ctrl+Shift+Z` functionality. Double check your code and make sure your thoroughly test it.

Please refer to my previous comment. The intended functionality still does not work.

**Pull request closed without merging**